silo-to-silo/configs/aurora-testnet.json

```diff
@@ -0,0 +1,7 @@
+{
+    "siloAccountId": "aurora",
+    "nativeTokenAccountId": "aurora",
+    "auroraSdkAddress": "0x425cA8f218784ebE2df347E98c626094B63E7f30",
+    "auroraUtilsAddress": "0xc129336a6995F3b70A7139585403B82098260172",
```

Comment:
Why we don't have "auroraCodecAddress", but we have this field in aurora-mainnet.json